src/cargo/ops/cargo_output_metadata.rs

```diff
@@ -22,16 +23,34 @@ pub struct OutputMetadataOptions<'a> {
 /// used versions - considering overrides - and writes all dependencies in a JSON
 /// format to stdout.
 pub fn output_metadata(opt: OutputMetadataOptions, config: &Config) -> CargoResult<ExportInfo> {
+    assert_eq!(opt.version, VERSION);
```

Comment:
Huh, I somehow thought that docopt should check the version automatically. Fixed.